help/export-your-organization.md

```diff
@@ -89,6 +98,37 @@ agreements and corporate policy for this type of export are in place. Note
 that many countries have laws that require employers to notify employees of
 their use of such an export.
 
+The output of this export is suitable to import into an on-premise
+installation of Zulip. It is not designed to be human-readable.
+
+## Compliance export
+
+{!owner-only.md!}
+
+This export is limited to paid Zulip Cloud Standard customers, though
+in rare cases may be available to other organizations in case of due
+legal process. As with full exports without member consent, you will
+also need to email us evidence that you have authority to read
```

Comment:
We don't use first person in this documentation (see https://zulip.readthedocs.io/en/latest/documentation/helpcenter.html#voice). So something like "email evidence... to Zulip support" works.

Reply:
This is also copied verbatim from the full export without member consent paragraph above it.

I'm not clear what change you're suggesting.

Reply:
I've stripped all of the "we" and "us" out of the document, including ones that predated this change.